demo/qat-init.sh

```diff
@@ -1,8 +1,56 @@
 #!/bin/sh -eu
 ENABLED_QAT_PF_PCIIDS=${ENABLED_QAT_PF_PCIIDS:-37c8 4940 4942}
 DEVS=$(for pf in $ENABLED_QAT_PF_PCIIDS; do lspci -n | grep -e "$pf" | grep -o -e "^\\S*"; done)
+SERVICES_LIST="sym;asym dc"
+QAT_4XXX_DEVICE_PCI_ID="0x4940"
+QAT_401XX_DEVICE_PCI_ID="0x4942"
 
-for dev in $DEVS; do
+check_config() {
```

Comment:
you're copying from qatlib's code without adding any reference to where this came from. our `qat-init.sh` is becoming same as what qatlib has so please evaluate can we just `wget/curl` that into our initcontainer and use that instead of maintaining our own version.

Reply:
Sure.! It was actually what I was thinking.!

Reply:
We will have this later.